xbmc/platform/win10/Win10App.cpp

```diff
@@ -19,50 +19,46 @@
  */
 
 #include "pch.h"
+#include "Win10App.h"
 
 #include "Application.h"
 #include "AppParamParser.h"
+#include "platform/Environment.h"
 #include "platform/xbmc.h"
 #include "platform/win32/CharsetConverter.h"
 #include "rendering/dx/RenderContext.h"
 #include "settings/AdvancedSettings.h"
-#include "platform/Environment.h"
 #include "utils/log.h"
 #include "utils/SystemInfo.h"
 #include "windowing/win10/WinEventsWin10.h"
-#include "Win10App.h"
 
-#include <agile.h>
-#include <collection.h>
 #include <ppltasks.h>
+#include <winrt/Windows.Storage.AccessCache.h>
 
-using namespace KODI::PLATFORM::WINDOWS10;
-using namespace Windows::ApplicationModel;
-using namespace Windows::ApplicationModel::Core;
-using namespace Windows::ApplicationModel::Activation;
-using namespace Windows::Foundation;
-using namespace Windows::Storage::AccessCache;
-using namespace Windows::UI::Core;
-using namespace Windows::UI::ViewManagement;
-
-IFrameworkView^ ViewProvider::CreateView()
-{
-  return ref new App();
+namespace winrt {
+  using namespace Windows::Foundation;
 }
+using namespace winrt::Windows::ApplicationModel;
+using namespace winrt::Windows::ApplicationModel::Core;
+using namespace winrt::Windows::ApplicationModel::Activation;
+using namespace winrt::Windows::Storage::AccessCache;
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Windows::UI::ViewManagement;
+using namespace KODI::PLATFORM::WINDOWS10;
 
 App::App()
 {
 }
 
 // The first method called when the IFrameworkView is being created.
-void App::Initialize(CoreApplicationView^ applicationView)
+void App::Initialize(CoreApplicationView const& applicationView)
```

Comment:
the hotly talked about east const :)
Doesn't really follow our coding style as we put it on the left.

Reply:
will fix